.gitignore

```diff
@@ -17,6 +17,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+.vscode
```

Comment:
I don't get why it committed your settings? like our setup must be different cause I don't think I have a file like this

Reply:
when i added the formatOnSave for prettier vscode env settings created this, thats why i added the .vscode to gitignore